salt/_pillar/metalk8s.py

```diff
@@ -168,6 +168,7 @@ def _load_addons(config_data):
     addons_data.setdefault("dex", {}).setdefault("enabled", True)
     addons_data.setdefault("loki", {}).setdefault("enabled", True)
     addons_data.setdefault("fluent-bit", {}).setdefault("enabled", True)
+    addons_data.setdefault("cert-manager", {}).setdefault("enabled", True)
```

Comment:
Let's remove this flag to me it does not need to be configurable since at some point it will be mandatory for other components